layouts/partials/list-posts.html

```diff
@@ -1,5 +1,10 @@
 <article class="post-title">
     <a href="{{ .Permalink }}" class="post-link">{{ .Title }}</a>
     <div class="flex-break"></div>
-    <span class="post-date">{{ dateFormat "January 2, 2006" .Date }}</span>
+    <span class="post-date">
+        {{ if .Site.Params.dateFormat }} 
+            {{ dateFormat .Site.Params.dateFormat .Date }}
+        {{ else }}
+            {{ dateFormat "January 2, 2006" .Date }} 
+        {{ end }} </span>
```

Comment:
```suggestion
    <span class="post-date">{{ dateFormat (or .Site.Params.dateFormat "January 2, 2006") }}</span>
```

I think this looks much cleaner, can you try with this

Reply:
> I think this looks much cleaner, can you try with this

too true, I have updated the code